src/Extensions/Hosting.Services.Web/HostBuilderExtensions.cs

```diff
@@ -13,6 +13,7 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.Omex.Extensions.Abstractions;

```

Comment:
is this needed?

Reply:
Yes, it is used in `BuilStatelesService` method